stream-projects/chat-theme/shared/alerts/signals.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  gql,
+  TruffleGQlConnection,
+  useSubscriptionSignal,
+} from "../../deps.ts";
+import { Alert } from "./types.ts";
+
+export type AlertConnection = TruffleGQlConnection<
+  Alert
+>;
+export const ALERTS_SUBSCRIPTION = gql<{ alertConnection: AlertConnection }>`
+  subscription AlertsReadyByTypeSubscription($types: [String], $status: String, $limit: Int) {
+    alertConnection(input: { types: $types, status: $status }, first: $limit) {
+      nodes {
+        id
+        orgId
+        type
+        status
+        sourceType
+        data
+      }
+    }
+  }
+`;
+
+export function useAlertSubscription$(
+  { status, types, limit }: {
+    status?: string;
+    types?: string[];
+    limit?: number;
+  },
+) {
+  const { signal$: alertConnection$ } = useSubscriptionSignal(
```

Comment:
we should standardize between ___Signal() vs ____$()

Reply:
yeah that's a good point, is seems like we usually use ` ___Signal()` for shared utils/hooks but `___$()` for instances but it gets a little grey for custom hooks.